sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,6 +181,32 @@ def test_spectral_embedding_amg_solver(seed=36):
     assert _check_with_col_sign_flipping(embed_amg, embed_arpack, 0.05)
 
 
+def test_spectral_embedding_amg_solver_failure(seed=36):
+    # Test spectral embedding with amg solver failure
```

Comment:
I think it remains hard for the reader to tell what case exactly is being tested here. At a minimum, it would be helpful to reference the issue where this bug is described

Reply:
@jnothman I agree, I'll see what I can do to clean this up based on your comments.

Reply:
Original bug reference added